WasatchNET/IEEPROM.cs

```diff
@@ -42,7 +42,7 @@ public interface IEEPROM
 
         // These will come with ENG-0034 Rev 4
         ushort startupIntegrationTimeMS { get; set; }
-        short startupDetectorTemperatureDegC { get; set; }
+        short TECSetpoint { get; set; }

```

Comment:
Personally I would have gone with "DetectorTECSetpoint" instead of TECSetpoint, since we also have to keep track of LaserTECSetpoint.

Reply:
The ambiguity is intentional since we are using the same field for both right now

Reply:
Ouch. I guess I deserved that :-)